cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -208,6 +209,12 @@ func (cmd *command) deploy(ctx context.Context) error {
 	}
 	clusterAccess.Successf("Successfully connected to cluster")
 
+	if !cmd.opts.CI && !cmd.opts.NonInteractive {
```

Comment:
The legacy logic for detecting managed runtime is already configured in line 129. At the time the CLI should use both approaches. In the future we'll remove the legacy `reconciler-based` approach.

Reply:
@Tomasz-Smelcerz-SAP this is the `Kyma alpha deploy` not the `Kyma deploy` command

Reply:
You are right. I've looked up in the `deploy` command file.
I think it's correct.